### PR TITLE
Unify typography, fixes #36

### PR DESCRIPTION
**Pull request by lwe**
See #36 for details.

@marcopluess there are multiple references to `h1` in `_default.scss`, any chance we can unify that into a single `h1`?

### CODE REVIEW

**Comment by marcopluess** [comment]
@lwe: like this?

File: app/assets/stylesheets/mtl/layouts/_default.scss [hunk 1]
```diff
@@ -21,6 +21,21 @@
...
+    margin: 0;
+    height: 64px;
+    line-height: 64px;
+    padding: 0 1.5rem;
+
+    @media #{$medium-and-down} {
+      padding-left: .5rem;
+    }
+    @media #{$small-and-down} {
+      line-height: 56px;
```
Comment (lwe) [inline app/assets/stylesheets/mtl/layouts/_default.scss]: do we also require height: 56px here?

Comment (marcopluess) [inline app/assets/stylesheets/mtl/layouts/_default.scss]: yes, as materializer adjusts the line-heigt of the nav icons.

**Comment by lwe** [comment]
👍 danke, will merge once the build is finished

**Pull request merged**